lerobot/scripts/train.py

```diff
@@ -241,6 +241,7 @@ def train(cfg: DictConfig, out_dir: str | None = None, job_name: str | None = No
         raise NotImplementedError()
 
     init_logging()
+    logging.info(cfg)
```

Comment:
This _could_ be useful, but right now it's just a massive one line json that's not very human readable. I think if we're going to do this it should be easier on the eyes.

Reply:
A very good suggestion. Is there any way to print it in a human readable way? Can I still use `logging`? @alexander-soare